### PR TITLE
Update kafka.yaml.erb

### PR DESCRIPTION
**Pull request by landau**
I copied this directly from https://github.com/DataDog/puppet-datadog-agent/blob/master/templates/agent-conf.d/kafka.yaml.erb and then made some adjustments based on the existing config that I found from old commits

There are many useful metrics for Kafka that are missing from the existing configuration.

### CODE REVIEW

File: templates/agent-conf.d/kafka.yaml.erb [hunk 1]
```diff
@@ -6,48 +6,52 @@
...
+  <%- if !server['tags'].nil? && server['tags'].any? -%>
     tags:
-<%- @tags.each do |tag_name, tag_value| -%>
-      <%= tag_name %>: <%= tag_value %>
-      environment: <%= @environment %>
-      app: <%= app %>
-<% end -%>
+    <%- server['tags'].each do |tag| -%>
+      - <%= tag %>
+      - environment: <%= @environment %>
```
Comment (landau) [inline templates/agent-conf.d/kafka.yaml.erb]: Added this line and the next based on existing config.

File: templates/agent-conf.d/kafka.yaml.erb [hunk 2]
```diff
@@ -6,48 +6,52 @@
...
-      environment: <%= @environment %>
-      app: <%= app %>
-<% end -%>
+    <%- server['tags'].each do |tag| -%>
+      - <%= tag %>
+      - environment: <%= @environment %>	  
+      - app: <%= app %>
+    <%- end -%>
+  <%- end -%>
+  <%- if !server['user'].nil? -%>
```
Comment (landau) [inline templates/agent-conf.d/kafka.yaml.erb]: changed this from `username` (https://github.com/DataDog/puppet-datadog-agent/blob/master/templates/agent-conf.d/kafka.yaml.erb#L19-L21) to `user` based on existing config.

**Comment by landau** [comment]
Thank you for merging this! :D

**Comment by landau** [comment]
How do we deploy this change?

**Comment by gloria-willadsen-brandorr** [comment]
Puppet may automatically pull this in when it runs. I'm checking with Brian to be sure.

**Comment by gloria-willadsen-brandorr** [comment]
Brian's going to show me how to take care of this. Please give me a day or so to take care of this. Brian is swamped right now.

**Comment by landau** [comment]
> Puppet may automatically pull this in when it runs

I don't see any new metrics in DD yet. Perhaps puppet is disabled on the Kafka machines.

Thanks, @gloria-willadsen-brandorr !

**Pull request merged**